test/extended/networking/ipsec.go

```diff
@@ -193,22 +193,19 @@ func ensureIPsecDisabled(oc *exutil.CLI) error {
 					return false, err
 				}
 			}
+			if done {
```

Comment:
Would be good to follow the same order here and in `ensureIPsecEnabled` on the machine config and cluster operator checks.

Also, I wouldn't reuse `ensureIPsecMachineConfigRolloutComplete` in `ensureIPsecEnabled`. Just call `areMachineConfigPoolsReadyWithIPsec` directly.

Reply:
sure, done.